packages/expo/src/Pedometer.android.ts

```diff
@@ -1,44 +0,0 @@
-import invariant from 'invariant';
```

Comment:
I forget, did you port this over?

Reply:
I compared the implementation with the one we had in expo-sensors and it wasn't clear to me why this one still exists, for example it depends on the existence of native methods that aren't available, so I deleted it. If people follow the Pedometer docs right now they will be using the expo-sensors version rather than this thing exported from the Expo package. So for these reasons it seems safe to delete.